### PR TITLE
" Adding Front End Data "

### PR DESCRIPTION
**Pull request by anudeepannin**

### CODE REVIEW

File: PostWebApllication/src/App.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  margin-left: 5px !important;
+}
+.btn-block {
+  width: 20% !important; 
+  margin-top:15px ;
+  margin-bottom: 15px;
+
+}
+
+
```
Comment (00kailash) [inline PostWebApllication/src/App.css]: Please remove the extra lines.

File: PostWebApllication/src/Comments.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    componentDidMount() {
+        this.refreshList();
+    }
+
+    changeCommentText = (e) => {
+        this.setState({ CommentText: e.target.value });
+    }
+
+    addComments(PostID) {
+        debugger;
```
Comment (00kailash) [inline PostWebApllication/src/Comments.js]: Remove the debugger from every file.

File: PostWebApllication/src/ListOfPosts.js [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+        })
+            .then(res => res.json())
+            .then((result) => {
+                alert(result);
+                this.refreshList();
+            }, (error) => {
+                alert('Failed');
+            })
+    }
+    LikeClick(row) {
```
Comment (00kailash) [inline PostWebApllication/src/ListOfPosts.js]: Always use the naming convention for all your code. For example:

1. Variable and function names are written as **camelCase**
2. Global variables are written in **UPPERCASE** 
3. Constants (like PI) written in **UPPERCASE**

File: PostWebApllication/src/App.css (hunk 1 above)
Comment (anudeepannin) [reply PostWebApllication/src/App.css]: I have removed the extra lines

File: PostWebApllication/src/Comments.js (hunk 2 above)
Comment (anudeepannin) [reply PostWebApllication/src/Comments.js]: removed the debugger

File: PostWebApllication/src/ListOfPosts.js (hunk 3 above)
Comment (anudeepannin) [reply PostWebApllication/src/ListOfPosts.js]: i have used  naming convention where i required

**Review by 00kailash — APPROVED** [review]

**Pull request merged**